go-controller/pkg/ovn/ovn.go

```diff
@@ -565,10 +561,11 @@ func (oc *Controller) WatchNodes() error {
 		AddFunc: func(obj interface{}) {
 			node := obj.(*kapi.Node)
 			if noHostSubnet := noHostSubnet(node); noHostSubnet {
-				oc.lsMutex.Lock()
-				defer oc.lsMutex.Unlock()
 				//setting the value to nil in the cache means it was not assigned a hostSubnet by ovn-kube
-				oc.logicalSwitchCache[node.Name] = nil
+				err := oc.lsCache.AddToLogicalSwitchCache(node.Name, nil)
```

Comment:
Code comment is no longer accurate. LS cache is now creating an object in the cache for the switch, just not setting any hostSubnets in it.

Reply:
Ack, will reword the comment to reflect the new logic.